packages/flutter/test/services/platform_channel_test.dart

```diff
@@ -231,10 +52,12 @@ void main() {
       BinaryMessages.setMockMessageHandler(
         'ch',
         (ByteData message) async {
-          final Map<dynamic, dynamic> methodCall = jsonMessage.decodeMessage(message);
+          final Map<dynamic, dynamic> methodCall =
+              jsonMessage.decodeMessage(message);
           if (methodCall['method'] == 'listen') {
             final String argument = methodCall['args'];
-            emitEvent(jsonMethod.encodeErrorEnvelope(code: '404', message: 'Not Found.', details: argument));
+            emitEvent(jsonMethod.encodeErrorEnvelope(
+                code: '404', message: 'Not Found.', details: argument));
```

Comment:
please don't make changes like this, this wrapping style violates our style guide